browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -1712,22 +1712,36 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, CosmeticFilteringSimple) {
 }
 
 // Test cosmetic filtering ignores content determined to be 1st party
-// This is disabled due to https://github.com/brave/brave-browser/issues/13882
-#define MAYBE_CosmeticFilteringProtect1p DISABLED_CosmeticFilteringProtect1p
-IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, MAYBE_CosmeticFilteringProtect1p) {
-  UpdateAdBlockInstanceWithRules("b.com##.fpsponsored\n");
+IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, CosmeticFilteringProtect1p) {
+  UpdateAdBlockInstanceWithRules(
+      "appspot.com##.fpsponsored\n"
+      "appspot.com##.fpsponsored1\n"
+      "appspot.com##.fpsponsored2\n"
+      "appspot.com##.fpsponsored3\n"
+      "appspot.com##.fpsponsored4\n");
 
   WaitForBraveExtensionShieldsDataReady();
 
-  GURL tab_url =
-      embedded_test_server()->GetURL("b.com", "/cosmetic_filtering.html");
+  GURL tab_url = embedded_test_server()->GetURL("test.lion.appspot.com",
```

Comment:
I would explicitly explain in a comment why appspot.com is meaningful here

Reply:
Done.